modules/gateways/monero/createinvoice.php

```diff
@@ -39,15 +41,16 @@ function monero_payment_id(){
 $array_integrated_address = $monero_daemon->make_integrated_address($payment_id);
 $address = $array_integrated_address['integrated_address'];
 $uri  =  "monero:$address?amount=$amount_xmr";
-
+$url  = Capsule::table('tblconfiguration')->where('setting', 'SystemURL')->first();
+$system_url = rtrim($url->value, '/');  // Strips default trailing / if there
 $secretKey = $GATEWAY['secretkey'];
 $hash = md5($invoice_id . $payment_id . $amount_xmr . $secretKey);
-echo "<link href='/modules/gateways/monero/style.css' rel='stylesheet'>";
-echo  "<script src='https://code.jquery.com/jquery-3.2.1.min.js'></script>";
-echo  "<script src='/modules/gateways/monero/spin.js'></script>";
+echo "<link href='$system_url/modules/gateways/monero/style.css' rel='stylesheet'>";
+echo  "<script src='https://code.jquery.com/jquery-3.6.0.min.js'></script>";
```

Comment:
Updated from the really old jquery which has vulns:
https://snyk.io/test/npm/jquery/3.2.1

Reply:
Thanks 👍🏻